src/main/java/org/folio/marc4ld/service/ld2marc/resource/Resource2MarcRecordMapperImpl.java

```diff
@@ -54,6 +65,7 @@ public Record toMarcRecord(Resource resource) {
       .sorted(comparing(VariableField::getTag))
       .forEach(marcRecord::addVariableField);
     addInternalIds(marcRecord, resource);
+    addDateFields(marcRecord, resource);
```

Comment:
minor: I think we don't need `addDateFields` method. We can directly invoke addUpdatedDateField from here.